### DISCUSSION TITLE
Is it possible to specify the name of the localdisk in a PVC

### DISCUSSION THREAD

**Discussion by wokalski** [hwameistor/hwameistor#1411] [Q&A]
I have a localdisk owned by `local-disk-manager`. I would like to create PVCs that explicitly assign particular disks to them. Is it possible?

**Comment by wokalski** [comment]
Also, im wondering if it's possible to say in PVC I want entire disk of this "class". For example I have some particular disks on which I'd like to run minio. They run on nodes labelled as "minio" but those machines have several disks attached and I want to run minio on disks with a particular `modelName` for instance.

**Reply by SSmallMonster** [reply]
It may not be possible to specify the disk directly inside the PVC. You need to create an LocalDiskClaim(LDC) first to expand the storage pool for that node. Then create a `StorageClass` of this type and specify, for example, hwameistor-local-disk-hdd inside the PVC.

See https://hwameistor.io/docs/nodes_and_disks/disk_nodes

**Reply by SSmallMonster** [reply]
>run minio on disks with a particular modelName for instance

we donot support model as a disk seclection now, but you can use `localDiskNames` directly in LocalDiskClaim to use the disks you  want to run minio.

**Reply by wokalski** [reply]
ok, so I guess I'd like a particular local-disk-claim to result in a "custom" storage class as a new functionality. Does it make sense to you?
